src/styles/globalStyle.ts

```diff
@@ -0,0 +1,42 @@
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+  :root {
+    --color-layer-0: ${({ theme }) => theme.layer0};
+    --color-layer-1: ${({ theme }) => theme.layer1};
+    --color-layer-2: ${({ theme }) => theme.layer2};
+    --color-layer-3: ${({ theme }) => theme.layer3};
+    --color-layer-4: ${({ theme }) => theme.layer4};
+    --color-layer-5: ${({ theme }) => theme.layer5};
+    --color-layer-6: ${({ theme }) => theme.layer6};
+    --color-layer-7: ${({ theme }) => theme.layer7};
+
+    --color-border: ${({ theme }) => theme.borderDefault};
+    --color-border-white: ${({ theme }) => theme.borderButton};
+    --color-border-red: ${({ theme }) => theme.borderDestructive};
+
+    --color-text-0: ${({ theme }) => theme.textTertiary};
+    --color-text-1: ${({ theme }) => theme.textSecondary};
+    --color-text-2: ${({ theme }) => theme.textPrimary};
```

Comment:
will rename these text + border variables in a follow up

Reply:
After:
<img width="157" alt="Screen Shot 2024-01-24 at 9 29 48 AM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/e567fb40-223e-453b-aa08-fb4bb1dd92de">

Before:
<img width="106" alt="Screen Shot 2024-01-24 at 9 30 23 AM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/2d3ba458-5ddc-45f5-bcff-5ef5746affbc">

I think the wrong value is being used for the border on the Button component.

Reply:
oh oops, my bad - it's missing transparency! Updated - ty!